include/ignition/physics/RemoveEntities.hh

```diff
@@ -76,8 +76,42 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature removes a nested Model entity from the
+    /// specified parent Model.
+    class IGNITION_PHYSICS_VISIBLE RemoveNestedModelFromModel
+        : public virtual Feature
+    {
+      public: template <typename PolicyT, typename FeaturesT>
+      class Model : public virtual Feature::Model<PolicyT, FeaturesT>
+      {
+        /// \brief Remove a Model that exists within this World.
+        /// \param[in] _index
+        ///   Index of the model within this world.
+        /// \return True if the model was found and removed.
+        public: bool RemoveModel(std::size_t _index);
+
+        /// \brief Remove a Model that exists within this World.
+        /// \param[in] _name
+        ///   Name of the model within this world.
+        /// \return True if the model was found and removed.
+        public: bool RemoveModel(const std::string &_name);
+      };
+
+      public: template <typename PolicyT>
+      class Implementation : public virtual Feature::Implementation<PolicyT>
+      {
+        public: virtual bool RemoveNestedModelByIndex(
+            const Identity &_worldID, std::size_t _modelIndex) = 0;
+
+        public: virtual bool RemoveNestedModelByName(
+            const Identity &_worldID, const std::string &_modelName) = 0;
+      };
+    };
+
     using RemoveEntities = FeatureList<
-      RemoveModelFromWorld
+      RemoveModelFromWorld,
+      RemoveNestedModelFromModel
```

Comment:
I think expanding a feature list is not very backwards-compatible, right? Let's try to get this in soon after the 4.0.0 release.

Reply:
This is actually already in #230. https://github.com/ignitionrobotics/ign-physics/pull/230/files#diff-ae51f2acfb9e12fdbdaf3388f166e00ae975bf14f063ddf7d2b2c5d4efc93c40R114. I started out this chain of PRs in my fork, so there was no way for me to set the target branches to my other PRs.

Reply:
Ahh perfect!